extensions/roblox/package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "roblox",
+  "title": "Roblox",
+  "description": "Raycast Extension for finding games, users & more on Roblox",
+  "icon": "extension-icon.png",
+  "author": "iamEvan",
+  "categories": [
+    "Productivity",
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "get-user-with-username",
+      "title": "Get User with Username",
+      "description": "Show a Roblox User with their username!",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "username",
+          "placeholder": "Username",
+          "type": "text",
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "get-user-with-id",
+      "title": "Get User with ID",
+      "description": "Show a Roblox User with their User ID!",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "id",
+          "placeholder": "User ID",
+          "type": "text",
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "get-my-account",
+      "title": "Get My Info",
+      "description": "Get your Roblox Account info!",
+      "mode": "view"
+    },
+    {
+      "name": "get-game-with-id",
+      "title": "Get Game with ID",
+      "description": "Show a Roblox Game with their ID!",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "placeId",
+          "placeholder": "Place ID",
+          "type": "text",
+          "required": false
+        },
+        {
+          "name": "universeId",
+          "placeholder": "Universe ID",
+          "type": "text",
+          "required": false
+        }
+      ]
+    },
+    {
+      "name": "search-games",
+      "title": "Search for Games",
+      "description": "Search for a Roblox Game!",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "startingQuery",
+          "placeholder": "Query",
+          "type": "text",
+          "required": false
+        }
+      ]
+    }
+  ],
+  "preferences": [
+    {
+      "name": "userId",
+      "title": "Your User ID",
+      "description": "Your Roblox User ID",
+      "type": "textfield",
+      "placeholder": "User ID",
+      "required": false
```

Comment:
```suggestion
      "required": true
```

Reply:
This should not be required, as it is only required for one command. The other commands can function without it.

Reply:
You can set `preferences:[]` directly for the given command instead of having them available for all commands